src/content/reference/react/experimental_taintUniqueValue.md

```diff
@@ -0,0 +1,162 @@
+---
+title: experimental_taintUniqueValue
+---
+
+<Wip>
+
+**This API is experimental and is not available in a stable version of React yet.**
+
+You can try it by upgrading React packages to the most recent experimental version:
+
+- `react@experimental`
+- `react-dom@experimental`
+- `eslint-plugin-react-hooks@experimental`
+
+Experimental versions of React may contain bugs. Don't use them in production.
+
+This API is only available inside React Server Components.
+
+</Wip>
+
+
+<Intro>
+
+`taintUniqueValue` lets you prevent a passing a password, crypto graphic key or token, as a string, bigint or typed array, to a Client Component. It blocks this string completely so it cannot be a value that's also used in other contexts. It needs to be unique.
```

Comment:
nit: "It blocks this string completely" is a little confusing since you just mentioned in the previous sentance that the value can be a string, bigint or typed array.

maybe replace "passing a password, crypto graphic key or token, as a string, bigint or typed array," with "value like a password or token" since you already mention the data types in the reference for the "value" parameter?